### ISSUE TITLE
fitBounds is not defined.

### ISSUE THREAD

**Issue by ethanclevenger91** [xkjyeah/vue-google-maps#12]
Given:

```html
<gmap-map
      ref="map"
      :center="center"
    >
</gmap-map>
```

It doesn't mind:

```js
vm.$refs.map.resize()
```

But 

```js
vm.$refs.map.fitBounds(vm.bounds)
```

Doesn't work: `vm.$refs.map.fitBounds is not a function`

However, this works:

```js
vm.$refs.map.$mapObject.fitBounds(vm.bounds);
```

**Comment by xkjyeah** [comment]
Hmmm. That's interesting... As a temporary workaround could you simply call
map.$mapObject.fitBounds?

**Comment by ethanclevenger91** [comment]
Yeah, that's what I'm doing. I actually edited the original issue to show
that that still worked

**Comment by xkjyeah** [comment]
OK I'll fix it when I'm back from my holiday. I have no access to a laptop
at the moment.

**Comment by ethanclevenger91** [comment]
No hurry. Thanks for all your work on this library

**Comment by xkjyeah** [comment]
Fixed in 0.4.6. [Test](http://xkjyeah.github.io/vue-google-maps/basic-map-functions.html)
